Mail2Bug/WorkItemManagement/IcmWorkItemManager.cs

```diff
@@ -281,9 +282,37 @@ public void ModifyWorkItem(long workItemId, string comment, Dictionary<string, s
                 Origin = ToolName
             };
 
+            ApplyOverrides(ref incident, values);

```

Comment:
Added it into CreateWorkItem as well in 1cf4f8a383fe4e84e25bcd4750548abffe95214e, had to refactor slightly to get it to work for the different types, so added in a test for that and merged the other three tests into one.